### DISCUSSION TITLE
"Submit a Login Flow" experimental warning

### DISCUSSION THREAD

**Discussion by RomanHotsiy** [ory/kratos#3388] [Q&A] [answered]
Hi Ory team 🙌

I've noticed a small inconsistency in the docs and wanted to clarify it.

The "Submit a login flow" is marked as experimental in the [API docs](https://www.ory.sh/docs/kratos/reference/api#tag/frontend/operation/updateLoginFlow) and SDK.

But there is no warning in the Ory documentation here: https://www.ory.sh/docs/kratos/self-service/flows/user-login#client-side-browser-clients-ajax

Could someone clarify if this is just a mistake and it should not be marked as experimental anymore?

Or if it's still experimental does the potential breaking change include removal of this method (this would prevent us from using it 🙈) or the method will stay but the shape may change?

Thanks for your great work!

**UPD**: I'm interested in the AJAX/fetch usage of this method.

**Comment by aeneasr** [comment, marked as answer]
Thank you for letting us know and sorry for the late reply! I have adjusted the SDK messages, as those endpoints for sure are no longer experimental!

**Reply by aeneasr** [reply]
https://github.com/ory/kratos/pull/3406

**Reply by RomanHotsiy** [reply]
Thank you 🎉
